base/logger_external.go

```diff
@@ -46,7 +46,7 @@ func initExternalLoggers() {
 }
 
 func updateExternalLoggers() {
-	if consoleLogger != nil && consoleLogger.shouldLog(LevelDebug, KeyWalrus) {
+	if consoleLogger != nil && consoleLogger.shouldLog(context.Background(), LevelDebug, KeyWalrus) {
```

Comment:
Does it make sense to pass context.Background() to shouldLog when we don't have an actual context?   shouldLogDatabaseContext looks like it exits earliest if the context is nil - would it be preferable to leave as nil?